### PR TITLE
Remove winds

### PR DESCRIPTION
**Pull request by gtmgianni**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/winds.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask 'winds' do
-  version '0.3.0'
-  sha256 'f3d97376a9f060c6d4c1b3f32383332cfa4a96c294aed760306111759adc292b'
+  version '2.0.74'
+  sha256 'dd11b1a6b019c9437a8bf3597faafb9c77c6b7d67e95201205e72647e6a5256f'
 
-  # github.com/GetStream/Winds was verified as official when first introduced to the cask
-  url "https://github.com/GetStream/Winds/releases/download/v#{version}/Winds.zip"
+  # s3.amazonaws.com/winds was verified as official when first introduced to the cask
+  url "https://s3.amazonaws.com/winds-#{version.major_minor}-releases/releases/Winds-#{version}-mac.zip"
```
Comment (vitorgalvao) [inline Casks/winds.rb]: We can’t use this link, because it’s walled.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/winds.rb (hunk 1 above)
Comment (gtmgianni) [reply Casks/winds.rb]: Does it count as walled if you don't need to register "fully" (you just need to enter an email address, and the download begins)? If so, we should probably just remove this cask.

Comment (vitorgalvao) [reply Casks/winds.rb]: > Does it count as walled if you don't need to register "fully"

Yes. Even if it works with a fake email, it still counts as walled.

The point is to make every cask easily auditable by anyone, without having to resort to “tricks”.

**Comment by commitay** [comment]
Thanks!

**Comment by commitay** [comment]
Re-added in https://github.com/Homebrew/homebrew-cask/pull/48699 with less wall.

**Pull request merged**